mnd/mnd/pdf_exports/about_me.py

```diff
@@ -21,14 +23,152 @@ def generate_pdf_form_fields(registry, patient):
         'pMobile': patient.mobile_phone,
         'pEmail': patient.email,
     }
-    patient_address = patient.home_address
-    if patient_address:
-        data.update({
-            'pSuburb': patient_address.suburb,
-            'pAddress': patient_address.address,
-            'pPostcode': patient_address.postcode,
-            'pState': _STATE_VALUE_MAPPING.get(patient_address.state, 'QLD') if patient_address.state else '',
+
+
+def _generate_patient_address_fields(patient_address):
+    if not patient_address:
+        return {}
+    state_value_mapping = {
+        'AU-SA': 'SA',
+        'AU-ACT': 'ACT',
+        'AU-NT': 'NT',
+        'AU-TAS': 'TAS',
+        'AU-QLD': 'QLD',
+        'AU-WA': 'WA',
+        'AU-VIC': 'VIC'
+    }
+    return {
+        'pSuburb': patient_address.suburb,
+        'pAddress': patient_address.address,
+        'pPostcode': patient_address.postcode,
+        'pState': state_value_mapping.get(patient_address.state, 'QLD') if patient_address.state else '',
+    }
+
+
+def _generate_patient_insurance_fields(patient, insurance):
+
+    def ndis_plan_manager(input):
+        return input.capitalize() if input else ''
+
+    def ndis_full_name(insurance):
+        fn = insurance.ndis_coordinator_first_name
+        ln = insurance.ndis_coordinator_last_name
+        if fn and ln:
+            return f"{fn} {ln}"
+        elif fn:
+            return fn
+        elif ln:
+            return ln
+        return ''
```

Comment:
`return ' '.join(n for n in (fn, ln) if n)` instead of all the `if`s and `elif`s?

Reply:
Updated.